src/Components/Server/test/Circuits/CircuitRegistryTest.cs

```diff
@@ -34,8 +35,13 @@ public void Register_AddsCircuit()
         public async Task ConnectAsync_TransfersClientOnActiveCircuit()
         {
             // Arrange
-            var registry = CreateRegistry();
-            var circuitHost = TestCircuitHost.Create();
+            var circuitIdFactory = new CircuitIdFactory(Options.Create<CircuitOptions>(new CircuitOptions
+            {
+                CircuitIdProtector = new EphemeralDataProtectionProvider().CreateProtector("Test")
+            }));
```

Comment:
Should we extract this into a reusable method? like `TestCircuitIdFactory.Create()`? I could see this used in a bunch of places and it would pay to clean it up preemptively.